tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/MongocryptdProseTests.cs

```diff
@@ -0,0 +1,423 @@
+﻿/* Copyright 2010-present MongoDB Inc.
```

Comment:
let's not split these tests. There are too many helpers around this. Additionally they're defined in the same scope in the spec

Reply:
`[Trait("Category", "Mongocryptd")]` can be added for method too